python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -461,32 +461,32 @@ def process_graph(args):
 
 if __name__ == '__main__':
     argparser = argparse.ArgumentParser("Preprocess graphs")
-    argparser.add_argument("--conf_file", type=str, required=True,
+    argparser.add_argument("--conf-file", type=str, required=True,
                            help="The configuration file.")
-    argparser.add_argument("--num_processes", type=int, default=1,
+    argparser.add_argument("--num-processes", type=int, default=1,
                            help="The number of processes to process the data simulteneously.")
-    argparser.add_argument("--num_processes_for_nodes", type=int,
+    argparser.add_argument("--num-processes-for-nodes", type=int,
                            help="The number of processes to process node data simulteneously.")
-    argparser.add_argument("--num_processes_for_edges", type=int,
+    argparser.add_argument("--num-processes-for-edges", type=int,
                            help="The number of processes to process edge data simulteneously.")
-    argparser.add_argument("--output_dir", type=str, required=True,
+    argparser.add_argument("--output-dir", type=str, required=True,
                            help="The path of the output data folder.")
-    argparser.add_argument("--graph_name", type=str, required=True,
+    argparser.add_argument("--graph-name", type=str, required=True,
                            help="The graph name")
-    argparser.add_argument("--remap_node_id", action='store_true',
+    argparser.add_argument("--remap-node-id", action='store_true',
                            help="Whether or not to remap node IDs.")
-    argparser.add_argument("--add_reverse_edges", action='store_true',
+    argparser.add_argument("--add-reverse-edges", action='store_true',
                            help="Add reverse edges.")
-    argparser.add_argument("--output_format", type=str, default="DistDGL",
+    argparser.add_argument("--output-format", type=str, default="DistDGL",
                            help="The output format of the constructed graph.")
-    argparser.add_argument("--num_partitions", type=int, default=1,
+    argparser.add_argument("--num-partitions", type=int, default=1,
```

Comment:
both in DGL.distributed, and our own graph_partition.py, we use “--num-parts”. Can we have a unified argument of the number of partitions?

Maybe we can use "--num-parts" here too.